spec/ics-23-standard-accumulator/README.md

```diff
@@ -0,0 +1,228 @@
+---
+ics: 23
+title: Standard Accumulator
+stage: draft
+category: ibc-core
+author: Christopher Goes <cwgoes@tendermint.com>
+created: 2019-04-16
+modified: 2019-04-25
+---
+
+# Synopsis
+
+An *accumulator*, or *cryptographic accumulator*, is a construction that produces a succinct, binding commitment to a set or indexed vector of elements and short membership and/or non-membership proofs for any element in the set or vector.
```

Comment:
Is "commitment" a widely understood term in the literature or do we need to define it?

Reply:
Commitment is widely understood